checkstyle.xml

```diff
@@ -1,120 +0,0 @@
-<?xml version="1.0"?>

```

Comment:
Should bring this back with https://github.com/f2prateek/android-checkstyle-plugin